test/e2e/taints_test.go

```diff
@@ -26,26 +34,35 @@ func runTaintsUpgradeFlow(test *framework.ClusterE2ETest, updateVersion v1alpha1
 // remove a taint from a node
 // add a taint to a node which already has another taint
 // add a taint to a node which had no taints
-func TestVSphereKubernetes121TaintsWorkerNodeGroups(t *testing.T) {
-	worker0 := "worker-0"
-	worker1 := "worker-1"
-	worker2 := "worker-2"
+func TestVSphereKubernetes121Taints(t *testing.T) {
```

Comment:
Is the feature flag being removed with these changes? If not, you need to pass that in too right?

Reply:
yes true, good call

Reply:
done https://github.com/aws/eks-anywhere/pull/1254/commits/64727114bf2dbb536274b62fee8e4a91b08ef0d5